src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -557,6 +557,12 @@ bool Assembler::needs_rex2(Register reg1, Register reg2, Register reg3) {
   return rex2;
 }
 
+bool Assembler::needs_evex(XMMRegister reg1, XMMRegister reg2, XMMRegister reg3) {
```

Comment:
This is an ASSERT / DEBUG only method, correct? Do you want to `#ifdef ASSERT` it accordingly?